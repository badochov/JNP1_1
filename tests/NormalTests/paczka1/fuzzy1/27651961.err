Error in line 9: R6PA5vpW A297 29,3
Error in line 10: R6PA5vpW S577 48,5
Error in line 20: R6PA5vpW S725 78,5
Error in line 30: R6PA5vpW A297 18,9
Error in line 17: WzV7GI9 S725 81,4
