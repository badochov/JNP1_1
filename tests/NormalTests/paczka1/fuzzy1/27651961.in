WzV7GI9 S73 3,4
wpqM S725 30,6
?FrTGb
WzV7GI9 S73 96,4
WzV7GI9 S73 18,9
WzV7GI9 S73 21,7
DQsBINW S725 39,4
wpqM S725 77,7
R6PA5vpW A297 29,3
R6PA5vpW S577 48,5
wpqM S725 71,1
S73 S577 98,8
cySJrENB S577 44,9
wpqM S725 126,6
?
cySJrENB S577 208,8
WzV7GI9 S725 81,4
cySJrENB S73 49,1
cySJrENB S73 84,5
R6PA5vpW S725 78,5
cySJrENB A297 21,7
DQsBINW S725 42,2
cySJrENB A297 66,4
S73 S577 30,3
?FrTGb
?FrTGb
cySJrENB S73 81,3
S73 S577 61,8
?
R6PA5vpW A297 18,9

cySJrENB S73 356,1
R6PA5vpW S73 49,9
?
cySJrENB S725 66,0
?cySJrENB
WzV7GI9 A297 11,8
?S725
FrTGb S577 5,6
WzV7GI9 A297 13,5