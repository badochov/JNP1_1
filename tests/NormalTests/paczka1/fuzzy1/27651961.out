WzV7GI9 S 95,8
wpqM S 102,6
S73 95,8
S725 102,6
DQsBINW S 2,8
S73 S 68,5
WzV7GI9 S 95,8
cySJrENB A 44,7 S 199,3
wpqM S 102,6
S73 131,2
A297 44,7
S577 232,4
S725 105,4
DQsBINW S 2,8
S73 S 68,5
WzV7GI9 S 95,8
cySJrENB A 44,7 S 474,1
wpqM S 102,6
S73 406,0
A297 44,7
S577 232,4
S725 105,4
cySJrENB A 44,7 S 474,1
S725 105,4
